### PR TITLE
Edit Position and Program

### PR DESCRIPTION
**Pull request by VoTuan232**
Chỉnh chức năng edit position và edit program

### CODE REVIEW

File: app/Http/Requests/PositionFormRequest.php [hunk 1]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
     {
-        return [
-            'name' => 'required|min:2|max:255|unique:positions,name,' . $this->id,
+        $messages = [
+            'name.required' => __('Position') . __('Required'),
+            'name.string' => __('Position') . __('String'),
+            'name.unique' => __('Position') . __('Unique'),
         ];
+        
+        return  Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.IncorrectSingle</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 34</br>Language constructs must be followed by a single space; expected 1 space but found "··"

Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 34</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/PositionFormRequest.php [hunk 2]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+                'name' => 'required|string|unique:positions',
+            ], $messages
+        );
+    }
+
+    public static function rulesUpdate($request, $id)
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 40</br>Whitespace found at end of line

File: app/Http/Requests/PositionFormRequest.php [hunk 3]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+    {
+        $messages = [
+            'name.required' => __('Position') . __('Required'),
+            'name.string' => __('Position') . __('String'),
+            'name.unique' => __('Position') . __('Unique'),
+        ];
+
+        $position = Position::findOrFail($id);
+
+        if($position->name == $request->name){
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 50</br>Expected 1 space(s) after IF keyword; 0 found

Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 50</br>Expected 1 space(s) after closing parenthesis; found 0

File: app/Http/Requests/PositionFormRequest.php [hunk 4]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+            return  Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.IncorrectSingle</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 51</br>Language constructs must be followed by a single space; expected 1 space but found "··"

Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 51</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/PositionFormRequest.php [hunk 5]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+                    'name' => 'required|string',
+                ], $messages
+            );
+        } else if ($position->name != $request->name) {
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.ControlStructures.ElseIfDeclaration.NotAllowed</br>**Severity**: warning</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 55</br>Usage of ELSE IF is discouraged; use ELSEIF instead

File: app/Http/Requests/PositionFormRequest.php [hunk 6]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+            return  Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.IncorrectSingle</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 56</br>Language constructs must be followed by a single space; expected 1 space but found "··"

Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 56</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/ProgramRequest.php [hunk 7]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
     {
-        return [
-            'name' => 'required|unique:programs,name',
+        $messages = [
+            'name.required' => __('Program') . __('Required'),
+            'name.string' => __('Program') . __('String'),
+            'name.unique' => __('Program') . __('Unique'),
         ];
+        
+        return  Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.IncorrectSingle</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 34</br>Language constructs must be followed by a single space; expected 1 space but found "··"

Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 34</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/ProgramRequest.php [hunk 8]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+    {
+        $messages = [
+            'name.required' => __('Program') . __('Required'),
+            'name.string' => __('Program') . __('String'),
+            'name.unique' => __('Program') . __('Unique'),
+        ];
+
+        $program = Program::findOrFail($id);
+        
+        if($program->name == $request->name){
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterKeyword</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 50</br>Expected 1 space(s) after IF keyword; 0 found

Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.ControlStructures.ControlSignature.SpaceAfterCloseParenthesis</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 50</br>Expected 1 space(s) after closing parenthesis; found 0

File: app/Http/Requests/ProgramRequest.php [hunk 9]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+            return  Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.IncorrectSingle</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 51</br>Language constructs must be followed by a single space; expected 1 space but found "··"

Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 51</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/ProgramRequest.php [hunk 10]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+                    'name' => 'required|string',
+                ], $messages
+            );
+        } else if ($program->name != $request->name) {
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.ControlStructures.ElseIfDeclaration.NotAllowed</br>**Severity**: warning</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 55</br>Usage of ELSE IF is discouraged; use ELSEIF instead

File: app/Http/Requests/ProgramRequest.php [hunk 11]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+            return  Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.LanguageConstructSpacing.IncorrectSingle</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 56</br>Language constructs must be followed by a single space; expected 1 space but found "··"

Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 56</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/PositionFormRequest.php [hunk 12]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
     {
-        return [
-            'name' => 'required|min:2|max:255|unique:positions,name,' . $this->id,
+        $messages = [
+            'name.required' => __('Position') . __('Required'),
+            'name.string' => __('Position') . __('String'),
+            'name.unique' => __('Position') . __('Unique'),
         ];
+
+        return Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 34</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/PositionFormRequest.php [hunk 13]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+        $messages = [
+            'name.required' => __('Position') . __('Required'),
+            'name.string' => __('Position') . __('String'),
+            'name.unique' => __('Position') . __('Unique'),
+        ];
+
+        $position = Position::findOrFail($id);
+
+        if ($position->name == $request->name) {
+            return Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 51</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/PositionFormRequest.php [hunk 14]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+                    'name' => 'required|string',
+                ], $messages
+            );
+        } elseif ($position->name != $request->name) {
+            return Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 56</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/ProgramRequest.php [hunk 15]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
     {
-        return [
-            'name' => 'required|unique:programs,name',
+        $messages = [
+            'name.required' => __('Program') . __('Required'),
+            'name.string' => __('Program') . __('String'),
+            'name.unique' => __('Program') . __('Unique'),
         ];
+
+        return Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 34</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/ProgramRequest.php [hunk 16]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+        $messages = [
+            'name.required' => __('Program') . __('Required'),
+            'name.string' => __('Program') . __('String'),
+            'name.unique' => __('Program') . __('Unique'),
+        ];
+
+        $program = Program::findOrFail($id);
+        
+        if ($program->name == $request->name) {
+            return Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 51</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/ProgramRequest.php [hunk 17]
```diff
@@ -21,10 +23,40 @@ public function authorize()
...
+                    'name' => 'required|string',
+                ], $messages
+            );
+        } elseif ($program->name != $request->name) {
+            return Validator::make($request->all(), [
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.SpaceBeforeCloseBracket</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 56</br>Expected 0 spaces before closing bracket; newline found

File: app/Http/Requests/PositionFormRequest.php [hunk 18]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
     {
-        return [
-            'name' => 'required|min:2|max:255|unique:positions,name,' . $this->id,
+        $messages = [
+            'name.required' => __('Position') . __('Required'),
+            'name.string' => __('Position') . __('String'),
+            'name.unique' => __('Position') . __('Unique'),
         ];
+
+        return Validator::make($request->all(),
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 34</br>Opening parenthesis of a multi-line function call must be the last content on the line

File: app/Http/Requests/PositionFormRequest.php [hunk 19]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
+            [
+                'name' => 'required|string|unique:positions',
+            ], $messages
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.MultipleArguments</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 37</br>Only one argument is allowed per line in a multi-line function call

File: app/Http/Requests/PositionFormRequest.php [hunk 20]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
+        $messages = [
+            'name.required' => __('Position') . __('Required'),
+            'name.string' => __('Position') . __('String'),
+            'name.unique' => __('Position') . __('Unique'),
+        ];
+
+        $position = Position::findOrFail($id);
+
+        if ($position->name == $request->name) {
+            return Validator::make($request->all(),
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 52</br>Opening parenthesis of a multi-line function call must be the last content on the line

File: app/Http/Requests/PositionFormRequest.php [hunk 21]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
+                [
+                    'name' => 'required|string',
+                ], $messages
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.MultipleArguments</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 55</br>Only one argument is allowed per line in a multi-line function call

File: app/Http/Requests/PositionFormRequest.php [hunk 22]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
+            );
+        } elseif ($position->name != $request->name) {
+            return Validator::make($request->all(),
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 58</br>Opening parenthesis of a multi-line function call must be the last content on the line

File: app/Http/Requests/PositionFormRequest.php [hunk 23]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
+                [
+                    'name' => 'required|string|unique:positions',
+                ], $messages
```
Comment (framgiaciv3) [inline app/Http/Requests/PositionFormRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.MultipleArguments</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 61</br>Only one argument is allowed per line in a multi-line function call

File: app/Http/Requests/ProgramRequest.php [hunk 24]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
     {
-        return [
-            'name' => 'required|unique:programs,name',
+        $messages = [
+            'name.required' => __('Program') . __('Required'),
+            'name.string' => __('Program') . __('String'),
+            'name.unique' => __('Program') . __('Unique'),
         ];
+
+        return Validator::make($request->all(),
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 34</br>Opening parenthesis of a multi-line function call must be the last content on the line

File: app/Http/Requests/ProgramRequest.php [hunk 25]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
+            [
+                'name' => 'required|string|unique:programs',
+            ], $messages
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.MultipleArguments</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 37</br>Only one argument is allowed per line in a multi-line function call

File: app/Http/Requests/ProgramRequest.php [hunk 26]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
+        $messages = [
+            'name.required' => __('Program') . __('Required'),
+            'name.string' => __('Program') . __('String'),
+            'name.unique' => __('Program') . __('Unique'),
+        ];
+
+        $program = Program::findOrFail($id);
+        
+        if ($program->name == $request->name) {
+            return Validator::make($request->all(),
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 52</br>Opening parenthesis of a multi-line function call must be the last content on the line

File: app/Http/Requests/ProgramRequest.php [hunk 27]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
+                [
+                    'name' => 'required|string',
+                ], $messages
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.MultipleArguments</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 55</br>Only one argument is allowed per line in a multi-line function call

File: app/Http/Requests/ProgramRequest.php [hunk 28]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
+            );
+        } elseif ($program->name != $request->name) {
+            return Validator::make($request->all(),
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.ContentAfterOpenBracket</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 58</br>Opening parenthesis of a multi-line function call must be the last content on the line

File: app/Http/Requests/ProgramRequest.php [hunk 29]
```diff
@@ -21,10 +23,43 @@ public function authorize()
...
+                [
+                    'name' => 'required|string|unique:programs',
+                ], $messages
```
Comment (framgiaciv3) [inline app/Http/Requests/ProgramRequest.php]: **Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Methods.FunctionCallSignature.MultipleArguments</br>**Severity**: error</br>**File**: app/Http/Requests/ProgramRequest.php</br>**Line**: 61</br>Only one argument is allowed per line in a multi-line function call

File: app/Http/Controllers/Admin/PositionController.php [hunk 30]
```diff
@@ -156,20 +146,11 @@ public function edit($id)
     public function update(Request $request, $id)
     {
         $data = $request->all();
-
+
```
Comment (BunnyPi04) [inline app/Http/Controllers/Admin/PositionController.php]: xóa space dư

File: app/Http/Requests/PositionFormRequest.php [hunk 31]
```diff
@@ -21,10 +23,49 @@ public function authorize()
      *
      * @return array
      */
-    public function rules()
+    public static function rulesStore($request)
```
Comment (BunnyPi04) [inline app/Http/Requests/PositionFormRequest.php:26]: sao lại sửa thành static?

File: resources/js/position.js [hunk 32]
```diff
@@ -180,6 +179,7 @@ $(function() {
                     checked: checked,
                 },
                 success: function (res) {
+                    console.log(res);
```
Comment (BunnyPi04) [inline resources/js/position.js]: xóa console.log

File: app/Http/Requests/PositionFormRequest.php (hunk 31 above)
Comment (VoTuan232) [reply app/Http/Requests/PositionFormRequest.php:26]: để em gọi lại bên controller ạ

**Comment by VoTuan232** [comment]
https://edu-redmine.framgia.vn/issues/4529

**Pull request merged**